index.bs

```diff
@@ -14,4 +14,21 @@ Group: WebView-CG
 
 # Introduction # {#introduction}
 
-Hello world!
\ No newline at end of file
+This specification describes a content embedding API that satisfies some
+critical use cases that <{iframe}> does not support. This embedding environment
+should allow embedding all content, including content which <{iframe}> cannot
+embed, and provide embedding sites more controlover that embedded content.
+Additionally, embedding a site's content should be possible without express
+permission from the embedded site.
+
+Since this is a particularly powerful API, it use and availability makes an app
+a target of various types of hacking. As a result, this API is limited to use in
+[Isolated Web Applications](https://github.com/WICG/isolated-web-apps/blob/main/README.md)
```

Comment:
In the preview, this isn't correctly linked. Perhaps this needs to be a bikeshed manual biblio entry and linked from here.

Reply:
Thanks, I completely missed that this wasn't linked properly. Looks like Bikeshed supports some Markdown, but not this linking syntax.

Reply:
I did figure out how to add this as a biblio entry. Let me know which approach you prefer.

Reply:
Biblio looks great!